### ISSUE TITLE
PHP 8.2 Creation of dynamic property warning

### ISSUE THREAD

**Issue by kreativan** [wikimedia/less.php#102]
Hello,
With PHP 8.2 getting creation of dynamic property warnings on compile:

```
PHP Deprecated: Creation of dynamic property Less_Tree_Variable::$parensInOp is deprecated in [.../less.php/lib/Less/Parser.php:2463]
PHP Deprecated: Creation of dynamic property Less_Tree_Dimension::$parensInOp is deprecated in [.../less.php/lib/Less/Parser.php:2464]
PHP Deprecated: Creation of dynamic property Less_Tree_Media::$allExtends is deprecated in [.../less.php/lib/Less/Visitor/extendFinder.php:96]
PHP Deprecated: Creation of dynamic property Less_Tree_Directive::$allExtends is deprecated in [.../less.php/lib/Less/Visitor/extendFinder.php:105]
```

**Comment by Krinkle** [comment]
@kreativan Which version of wikimedia/less.php are you on? The latest version is v4.1.1. I believe these warnings were fixed in v3.2.0 in https://github.com/wikimedia/less.php/commit/a93b0827b650a347f7c694e1e234f8c86efa8812.

**Comment by kreativan** [comment]
@Krinkle Yep, you are right! I thought i had a latest version, i didn't have an absolute path when require autoloader so was using the older version even i renamed the folder...
Thanks!
